src/overlay/Peer.cpp

```diff
@@ -581,6 +582,10 @@ Peer::msgSummary(StellarMessage const& msg)
 
     case SURVEY_REQUEST:
     case SURVEY_RESPONSE:
+    case TIME_SLICED_SURVEY_REQUEST:
```

Comment:
I think it'd be helpful to articulate deprecation plan for old-style survey messages. Do we plan to keep that functionality around? If not, ideally, once the network switches to new survey, we can remove support for old-style survey completely. We should be able to remove XDR definitions for those as well: the benefit here is that old/deprecated messages will be rejected as early as possible by the XDR parser, and we don't need to worry about invalid messages accidentally slipping into core and not being handled correctly (something we got bitten by before). I see that current XDR extends the existing survey in a few places, so wanted to get your thoughts here.